lizmap/modules/lizmap/classes/qgisForm.class.php

```diff
@@ -452,13 +452,70 @@ public function setFormDataFromFields($feature)
                 }
                 $form->setData($ref.'_hidden', $value);
             } else {
+                if (in_array(strtolower($this->formControls[$ref]->fieldEditType), array('date', 'time', 'datetime'))) {
+                    $date = new DateTime($value);
```

Comment:
$value could be in the `$edittype->options->field_format`

Reply:
Here, `$value` is the date/time already stored in the DB that we want to put in the datepicker when we edit the feature and to do so, we need to convert it into the format accepted by Jelix.
[date](https://github.com/jelix/jelix/blob/8f313e8aa19f7a72bad7733a4f9354f5916b9ccb/lib/jelix/plugins/formwidget/date_html/date_html.formwidget.php#L79)
[datetimepicker](https://github.com/jelix/jelix/blob/8f313e8aa19f7a72bad7733a4f9354f5916b9ccb/lib/jelix/plugins/formwidget/datetime_html/datetime_html.formwidget.php#L89)